analysers/analyser_osmosis_powerline.py

```diff
@@ -681,50 +681,71 @@ def __init__(self, config, logger = None):
-In which case make use of the `disused:` [lifecycle prefix](https://wiki.openstreetmap.org/wiki/Lifecycle_prefix).'''))
+'''It's possible that being built or disused power features could be disconnected from the live network.
+In which case make use of the `construction:` or `disused:` [lifecycle prefixes](https://wiki.openstreetmap.org/wiki/Lifecycle_prefix).'''))
         self.classs[3] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:chair'],
             title = T_('Connection between different voltages'),
-            detail = T_('Two power lines meet at this point, but have inconsistent voltages (`voltage=*`).'),
+            detail = T_('Several power lines seem to connect at this point, but have inconsistent voltages (`voltage=*`).'),
             fix = T_(
 '''Check if the voltages are really different.
-Add a transformer using `power=transformer` (standalone transformers) or `power=pole + transformer=*` (pole mounted transformers).'''))
+If yes, add a substation (`power=substation`) or transformer (`power=transformer`) on this node (standalone transformers)
+or `power=pole + transformer=*` (pole mounted transformers).'''),
+            trap = T_(
+'''Some lines may share the same support without connection.
+Have a look to `line_management=termination` or `line_management=cross` for such sitations.'''))
         self.classs[4] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
             title = T_('Non power node on power way'),
             detail = T_(
-'''Power lines can only form a straight line between supports and therefore shouldn't
+'''Power lines can only form a straight line between their supports and therefore shouldn't
 have additional nodes that aren't tagged as a `power` feature.'''),
             fix = T_(
 '''If this node is a tower or pole, use the tag `power=tower` or
 `power=pole`. Otherwise remove it.'''))
         self.classs_change[5] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
             title = T_('Missing power tower or pole'),
             detail = T_(
-'''Based on the statistical frequency of the poles on this power line,
-there's likely an unmapped pole nearby.'''))
+'''Based on the statistical frequency of the supports on this power line,
+there's likely an unmapped pole nearby.'''),
+            fix = T_(
+'''Seek for possibly missing support in the surroundings and add it to the line.'''))
         self.classs[7] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:chair'],
-            title = T_('Unmatched voltage of line on substation'))
+            title = T_('Unmatched voltage of line on substation'),
+            detail = T_(
+'''A line terminates near of a substation with inconsistent voltages between them.'''),
+            fix = T_(
+'''Continue the line towards its actual endpoint, check voltages on line and on substation and adjust what is required to better reflect what is seen on ground.''')),
```

Comment:
[...] on the line and the substation and adjust what is necessary to reflect what is seen on the ground.